### DISCUSSION TITLE
How to edit a image and apply that image edited when click on "add media"

### DISCUSSION THREAD

**Discussion by duongductrong** [awcodes/filament-curator#362] [General]
I finding a action allows i using a edit image when I adding a new media

**Comment by awcodes** [comment]
Sorry, can you explain more. I'm just not understanding what you are asking.

**Comment by DanielSpravtsev** [comment]
@awcodes I think this is the same Issue. Hope you understood my explanation more :) https://github.com/awcodes/filament-tiptap-editor/issues/236
